dipy/reconst/shm.py

```diff
@@ -362,7 +362,7 @@ def sph_harm_ind_list(sh_order):
     n_range = np.arange(0, sh_order + 1, 2, dtype=int)
     n_list = np.repeat(n_range, n_range * 2 + 1)
 
-    ncoef = (sh_order + 2) * (sh_order + 1) / 2
+    ncoef = (sh_order + 2) * (sh_order + 1) // 2
```

Comment:
Yes - this should be integer division. It's the number of coefficients for the SH basis set -- always an integer. 

The errors on that other PR have to do with the fact that the call to `empty` below raises a warning otherwise ("using a non-integer number instead of an integer will result in an error in the future")